backend/src/main/java/com/google/minicrm/data/LeadStatus.java

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.minicrm.data;
+
+/**
+ * Represents all the possible statuses a lead can have throughout its lifecycle.
+ */
+public enum LeadStatus {
```

Comment:
this can be made more robust by adding explicit indices:

"public enum LeadStatus {
NEW(0), 
OPEN(1)
...
CLOSED_NOT_CONVERTED(4);

private final int index;

LeadStatus(int index) {
this.index = index
}

public int getIndex() {
return index;
}
}

Reply:
Will do!